clients/ctl/admin-ui/src/features/dataset/ClassifiedDataCategoryDropdown.tsx

```diff
@@ -0,0 +1,109 @@
+import { Box, ButtonProps } from "@fidesui/react";
+import { MultiValue, Select } from "chakra-react-select";
+
+import { DataCategory } from "~/types/api";
+
+import {
+  DataCategoryDropdown,
+  Props as DataCategoryDropdownProps,
+} from "./DataCategoryInput";
+
+// TODO: just making up a structure until we have something real from the API
```

Comment:
Nice. The category fides key will come from the [classification label](https://github.com/ethyca/fides/pull/1113/files#diff-d65a475197936464f11d722a344a559f2be72a46338a922105707f64d3f4bf40R31) and the score -> confidence. It should be easy enough in another PR to make a selector that combines these into this interface.